### PR TITLE
My Hello Microverse Project

### PR DESCRIPTION
**Pull request by Smart1-hub**
My first Github workflow project at Microverse, kindly help look through

### CODE REVIEW

File: .github/workflow/linters.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+.github/workflows/linters.yml
```
Comment (Amrhub) [inline .github/workflow/linters.yml]: - [ ] I know linters are confusing at the beginning, let's make sure we name the file path correctly we just have a typo it should be inside.github ✔️ and then create a folder names `workflows` plural we just missed the `s` In order for it to work on GitHub it should be in this directory `.github/workflows` then we create a file named linters.yml just like you did 👍🏻.
Then we paste the best linter config that suits our project https://github.com/microverseinc/linters-config from this link in our case would be `html-css` you find this file https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml you can copy its content in our linters.yml

File: README.md [hunk 2]
```diff
@@ -2,25 +2,12 @@
```
Comment (Amrhub) [inline README.md:2]: - [ ] I could see we added .gitignore, let's make sure we add `node_modules` and `test.md` as per project requirements. anything in this file will make git ignores it from tracking it. you could read more about it https://git-scm.com/docs/gitignore

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @Smart1-hub,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Smart1-hub — COMMENTED** [review]
The typo error in workflows has been corrected, prescribed code has been included in the linters file and node_modules file has been included. Thanks

File: README.md [hunk 3]
```diff
@@ -2,25 +2,12 @@
 
 # Project Name
```
Comment (OLIPLICHE) [inline README.md:3]: ## OPTIONAL

- [ ] Replace `project Name` with `Hello Microverse`

File: README.md [hunk 4]
```diff
@@ -42,8 +27,8 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
```
Comment (OLIPLICHE) [inline README.md:28]: ## OPTIONAL

- [ ] Please Replace `Author1` with your `full name`. 👍

File: README.md [hunk 5]
```diff
@@ -32,8 +19,6 @@ To get a local copy up and running follow these simple example steps.
 
 ### Usage
 
-### Run tests
-
 ### Deployment
```
Comment (OLIPLICHE) [inline README.md:22]: ## OPTIONAL

- [ ] Please remove unused sections from your README to look more professional. 👍

**Review by OLIPLICHE — APPROVED** [review]
Hi 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Smart1-hub** [comment]
Thank so much for the review. Please, the node_nodules, should that be a folder inside .github/workflows?
MURP, RTP(Urban Planner & GIS Consultant ) 

    On Tuesday, March 15, 2022, 02:15:33 PM GMT+1, Amr Ahmed ***@***.***> wrote:  
 
 


@Amrhub requested changes on this pull request.

Hi @Smart1-hub, 👋🏻

Good job so far! 👍🏻
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

To Highlight ✔️
 
Highly organized and clean code ✔️ 👨🏻‍💻
Professional documentation ✔️
Correct workflow ✔️
Meaningful commit messages ✔️
 
Required Changes ♻️
 
Check the comments under the review.


Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding! 👏 👏 👏
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

In .github/workflow/linters.yml:
> @@ -0,0 +1 @@
+.github/workflows/linters.yml
   
   -  I know linters are confusing at the beginning, let's make sure we name the file path correctly we just have a typo it should be inside.github ✔️ and then create a folder names workflows plural we just missed the s In order for it to work on GitHub it should be in this directory .github/workflows then we create a file named linters.yml just like you did 👍🏻.   
Then we paste the best linter config that suits our project https://github.com/microverseinc/linters-config from this link in our case would be html-css you find this file https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml you can copy its content in our linters.yml

In README.md:
> @@ -2,25 +2,12 @@
 
   
   -  I could see we added .gitignore, let's make sure we add node_modules and test.md as per project requirements. anything in this file will make git ignores it from tracking it. you could read more about it https://git-scm.com/docs/gitignore

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Pull request merged**